src/server/utilities/liteATIClickTracking/index.ts

```diff
@@ -0,0 +1,93 @@
+export default () => {
+  window.addEventListener('load', () => {
+    document.addEventListener('click', (event: MouseEvent) => {
+      const targetElement = event.target as HTMLElement;
+      // eslint-disable-next-line no-undef
+      if (targetElement?.tagName === 'A') {
+        event.stopPropagation();
+        event.preventDefault();
+
+        const atiURL = targetElement.getAttribute('data-lite-ati-tracking');
+
+        if (atiURL == null) {
+          return;
+        }
+
+        const currentAnchorElement = event.currentTarget as HTMLAnchorElement;
+        const nextPageUrl = currentAnchorElement?.href;
+
+        const {
+          screen: { width, height, colorDepth, pixelDepth },
+          innerWidth,
+          innerHeight,
+        } = window;
+        const now = new Date();
+        const hours = now.getHours();
+        const mins = now.getMinutes();
+        const secs = now.getSeconds();
+
+        // COOKIE SETTINGS
+        const cookieName = 'atuserid';
+        const expires = 397; // expires in 13 months
+        const cookiesForPage = `; ${document.cookie}`;
+        const atUserIdCookie = cookiesForPage.split(`; ${cookieName}=`);
+        let atUserIdValue = null;
+
+        if (atUserIdCookie.length === 2) {
+          const cookieInfo = atUserIdCookie.pop()?.split(';').shift();
+
+          if (cookieInfo) {
+            const decodedCookie = decodeURI(cookieInfo);
+            const user = JSON.parse(decodedCookie);
+            atUserIdValue = user.val;
+          }
+        }
+
+        if (!atUserIdValue && crypto.randomUUID) {
+          atUserIdValue = crypto.randomUUID();
+        }
+
+        const stringifiedCookieValue = JSON.stringify({ val: atUserIdValue });
+        if (atUserIdValue) {
+          document.cookie = `${cookieName}=${stringifiedCookieValue}; path=/; max-age=${expires}; Secure;`;
+        }
+
+        const rValue = [
```

Comment:
Can we rename this to describe what the r value represents? 

```suggestion
        const screenResolutionColourDepth = [
```